Dockerfile

```diff
@@ -10,19 +10,19 @@ ENV SHELL=/bin/sh \
    KEGBOT_ENV=debug
 
 RUN apk update && \
-    apk add --no-cache \
-      bash \
-      curl \
-      libjpeg \
-      libjpeg-turbo \
-      openjpeg && \
-   pip install pipenv
+   apk add --no-cache \
+   bash \
+   curl \
+   libjpeg \
+   libjpeg-turbo \
+   openjpeg && \
+   pip install pipenv==2018.11.26
 
 ADD Pipfile Pipfile.lock ./
 RUN apk add --no-cache mariadb-connector-c-dev libpq && \
    apk add --no-cache --virtual _build-deps \
-     build-base mariadb-dev postgresql-dev libjpeg-turbo-dev zlib-dev py-gevent libffi-dev && \
-   pipenv install --deploy --system && \
+   build-base mariadb-dev postgresql-dev libjpeg-turbo-dev zlib-dev py-gevent libffi-dev && \
+   pipenv install --deploy --system --skip-lock && \
```

Comment:
Ah, we don't want to `--skip-lock` as that would break version pinning / predictability of dependencies.

Will take a quick look at this locally & see if there's a better alternative.